Makefile

```diff
@@ -39,6 +39,12 @@ build-darwin: generate
 generate:
 	go generate ./...
 
+.PHONY: docs
+docs:
+	CGO_ENABLED=0 GOOGS=darwin go build -o ./bin/kyma-darwin-doc -ldflags '-X main.AutoGenDoc=true' ./cmd
```

Comment:
We can just do:
```bash
go run -ldflags '-X main.AutoGenDoc=true' ./cmd
```

Benefits:
- It works in any platform (current command only works on OSX).
- No need to crete a binary
- No need to delete it afterwards.

Reply:
Fixed

Reply:
great thanks